gladier/tools/__init__.py

```diff
@@ -3,10 +3,35 @@
 import typing as t
 
 from .helpers import exclusive_validator_generator, validate_path_property
+from .builtins import ChoiceState, ExpressionEvalState, FailState, WaitState, PassState
```

Comment:
I see you've taken the `Globus` prefix off of the various states, and that you're also raising these to the `gladier.tools` level rather than the sub-modules under there. I'm ok with that.

I guess we should be consistent on whether we want the State suffix. Compute still has it and the others do not. I see pros/cons to either. More concise without it, less chance to have rather generic class names like "Compute" or "Transfer" if we don't have it. I'm ok either way I think

Reply:
I also kinda like "Actions" if we are to use a suffix. Seems that the no suffix approach that you've put in place is good (enough?). Just changing the GlobusCompute one to get consistency seems like the remaining item here.